zk_toolbox/crates/zk_inception/src/commands/prover/protocol_version.rs

```diff
@@ -0,0 +1,44 @@
+use std::{fs, path::Path};
+
+use common::logger;
+use config::EcosystemConfig;
+use xshell::{cmd, Shell};
+
+pub async fn run(shell: &Shell) -> anyhow::Result<()> {
+    let link_to_code = EcosystemConfig::from_file(shell)?.link_to_code;
+
+    let protocol_version = get_protocol_version(shell, &link_to_code).await?;
+    let snark_wrapper = get_snark_wrapper(&link_to_code).await?;
+
+    logger::info(format!(
+        "Current protocol version found in zksync-era: {}, snark_wrapper: {}",
+        protocol_version, snark_wrapper
+    ));
+
+    Ok(())
+}
+
+async fn get_protocol_version(shell: &Shell, link_to_code: &Path) -> anyhow::Result<String> {
+    let path = link_to_code.join("prover/crates/bin/prover_version/Cargo.toml");
+    let protocol_version = cmd!(
+        shell,
+        "cargo +nightly-2024-08-01 run --manifest-path {path}"
```

Comment:
why do you need to specify the version? 
we will take the version from `rust-toolchain`

Reply:
and why do you need to specify the whole manifest? 
seems it's easier to `push_dir('prover')` and run `cargo run --bin prover_version`